.gitignore

```diff
@@ -58,3 +59,5 @@ node_modules
 /stash
 dist
 .DS_Store
+
+
```

Comment:
Idk why they kept being added... Sorry for the 2 extra line return at the end of the file